modules/combined/test/tests/frictional_contact/sliding_elastic_blocks_2d/tests

```diff
@@ -1,14 +1,20 @@
 [Tests]
-  [./base]
+  issues = '#6524'
+  design = 'syntax/Contact/index.md MechanicalContactConstraint.md'
+  [./sliding_elastic_blocks_2d]
     type = 'Exodiff'
     input = 'sliding_elastic_blocks_2d.i'
     exodiff = 'sliding_elastic_blocks_2d_out.e'
+    custom_cmp = 'sliding_elastic_blocks_2d.cmp'
     superlu = true
+    requirement = 'The contact system shall enforce 2D line contact between quads with significant accumulated slip. '
   [../]
-  [./tangential_penalty]
+  [./sliding_elastic_blocks_2d_tp]
     type = 'Exodiff'
     input = 'sliding_elastic_blocks_2d_tp.i'
-    exodiff = 'sliding_elastic_blocks_2d_tp_out.e'
+    exodiff = 'sliding_elastic_blocks_2d_tp_exodus.e'
+    custom_cmp = 'sliding_elastic_blocks_2d.cmp'
     superlu = true
+    requirement = 'The contact system shall enforce 2D line contact between quads with significant accumulated slip, when formulation selected is penalty. '
```

Comment:
penalty -> tangential_penalty
Also, remove the extra whitespace at the end of this requirement and the one above.